app/pages/lab/workflows.jsx

```diff
@@ -33,8 +34,9 @@ const WorkflowsPage = (props) => {
       <p>A workflow is the sequence of tasks that you’re asking volunteers to perform.</p>
       <p>An asterisk (*) denotes a default workflow.</p>
       <p>If you have multiple workflows you can rearrange the order in which they are listed on your project's front page by clicking the reorder view button and then clicking and dragging on the left gray tab next to each workflow title listed below.</p>
+      <p>NOTE: Please leave at least one active workflow; even if all workflows are 100% complete.</p>
```

Comment:
Perhaps bold lettering can be used for emphasis instead of capitals?

Reply:
Sure.